src/WebJobs.Script/Description/CSharp/CSharpFunctionInvoker.cs

```diff
@@ -31,6 +32,8 @@ public class CSharpFunctionInvoker : ScriptFunctionInvokerBase
         private readonly Collection<FunctionBinding> _inputBindings;
         private readonly Collection<FunctionBinding> _outputBindings;
         private readonly IFunctionEntryPointResolver _functionEntryPointResolver;
+        private readonly FunctionMetadata _functionMetadata;
```

Comment:
The function metadata was pushed down to the `ScriptFunctionInvokerBase` and exposed as a property.